src/analytics/parameter.ts

```diff
@@ -128,19 +128,19 @@ export function createAthenaStackParameters(scope: Construct): {
   const athenaEventTableParam = new CfnParameter(scope, 'AthenaEventTable', {
     description: 'The Athena event table name.',
     type: 'String',
-    default: TABLE_NAME_EVENT,
+    default: TABLE_NAME_EVENT_V2,
   });
 
-  const athenaEventParamTableParam = new CfnParameter(scope, 'AthenaEventParamTable', {
-    description: 'The Athena event table name.',
+  const athenaSessionTableParam = new CfnParameter(scope, 'AthenaSessionTable', {
```

Comment:
@tyyzqmf a parameter is renamed for console plane changes

Reply:
No impact, the backend did not set this parameter.